docs/core/rid-catalog.md

```diff
@@ -116,22 +116,24 @@ See [Prerequisites for .NET Core on Windows](windows-prerequisites.md) for more
 - Debian
   - `debian-x64`
   - `debian.8-x64`
+  - `debian.9-x64`
```

Comment:
This should have `(.NET Core 1.1 or later versions)`. We don't support `debian.9` in 1.0.

Reply:
Thanks @eerhardt. Fixed!